ssl/record/ssl3_record.c

```diff
@@ -1321,6 +1343,8 @@ int tls1_mac(SSL *ssl, SSL3_RECORD *rec, unsigned char *md, int sending)
     unsigned char header[13];
     int stream_mac = (sending ? (ssl->mac_flags & SSL_MAC_FLAG_WRITE_MAC_STREAM)
                       : (ssl->mac_flags & SSL_MAC_FLAG_READ_MAC_STREAM));
+    int tlstree_mac = (sending ? (ssl->mac_flags & SSL_MAC_FLAG_WRITE_MAC_TLSTREE)
+                      : (ssl->mac_flags & SSL_MAC_FLAG_READ_MAC_TLSTREE));
```

Comment:
please remove the outer parens here and align ? with :
I see you copied that style from above, please fix that as well,
thanks

Reply:
Done.